.storybook/test-runner.ts

```diff
@@ -0,0 +1,12 @@
+// TODO: Convert this to ESM once the bug is resolved
+// https://github.com/storybookjs/test-runner/issues/293
+
+const config = {
+  async postRender(page, _context) {
+    const elem = await page.$('#story-root');
```

Comment:
The `$` syntax, is this by any chance jQuery? :O

Reply:
It's not JQuery, It's the page object from Playwright.
https://playwright.dev/docs/api/class-page

I just read while checking that the `$` syntax is deprecated so I'll switch to their newer function called `locator`

Reply:
Thanks! Appreciated :) (Nothing against jQuery just surprised lol).

Reply:
Updated to `locator`